aerogear/mobile-cli
# [client provision via catalog] create client using service catalog

**Describe what this PR does and why we need it**:
- [x] provision via catalog

**Does this PR depend on another PR (Use this to track when PRs should be merged)**
https://github.com/aerogearcatalog/android-app-apb/pull/11

**Which issue this PR fixes (This will close that issue when PR gets merged)**
https://issues.jboss.org/browse/AEROGEAR-3023


## Verification steps*:
1. [x] pull this changes and build binary
2. [x] run different variants of `mobile create client ...`
3. [x] ensure that each mobile client has corresponding service instance
4. [x] MAR view annotated service_instance_id should match service instance id


*needs:
https://github.com/aerogearcatalog/android-app-apb/pull/11
https://github.com/aerogearcatalog/cordova-app-apb/pull/9
https://github.com/aerogearcatalog/ios-app-apb/pull/10
https://github.com/aerogearcatalog/xamarin-app-apb/pull/5
